recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ requirements:
     - cookiecutter >=1.6.0
     - diff-match-patch >=20181111
     - intervaltree >=3.0.2
-    - ipython >=7.31.1,<9.0.0,!=8.8.0,!=8.9.0,!=8.10.0,!=8.11.0,!=8.12.0,!=8.12.1
+    - ipython >=7.31.1,<9.0.0,!=8.8.0,!=8.9.0,!=8.10.0,!=8.11.0,!=8.12.0,!=8.12.1,!=8.17.1
```

Comment:
This actually needs to be `>=8.12.2,<9.0.0` because we don't support Python 3.7 anymore (a requirement on IPython 7 was necessary here due to that) and `8.12` was the last version with Python 3.8 support.